src/DashboardService.Models.Dto/Requests/Board/PatchBoardRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace LT.DigitalOffice.DashboardService.Models.Dto.Requests.Board;
+
+public class PatchBoardRequest
+{
+  public Guid Id { get; set; }
```

Comment:
is this really necessary? id is sent separately with this request

Reply:
+